src/components/publishing/sections/sections.tsx

```diff
@@ -34,10 +36,11 @@ const Sections: React.SFC<SectionsProps> = props => {
 function renderSections(article) {
   const sections = article.sections
   const renderedSections = sections.map((section, i) => {
-    const child = getSection(section)
+    const child = getSection(section, article.layout)
+    const layout = section.body ? findBlockquote(section.body, article.layout) : section.layout
```

Comment:
Yeah that's right we want `SectionContainer` to always set the section's width. Definitely 👍  for setting the layout for blockquote at the section level in Writer. Seems much cleaner. Soo for now we can keep this!